Sources/Core/aws-sdk-swift.AWSClientRuntime/Package.swift

```diff
@@ -2,6 +2,10 @@
 
 import PackageDescription
 
+let sdkVersion: Version = "0.0.15"
+let smithySwiftVersion: Version = "0.100.0"
+let crtVersion: Version = "0.32.0"
```

Comment:
The dependency versions have been extracted out to these variables.  Ignore the specific versions for now.